src/components/GameUI.js

```diff
@@ -13,7 +13,7 @@ const renderGame = (game) => {
         <p class="likes-quantity">0 likes</p>
       </div>
     </div>
-    <button class="btn btn-comments" type="button">Comments</button>`;
+    <button class="btn btn-comments" type="button" id=${game.id}>Comments</button>`;
```

Comment:
- [ ] We have used the same ID for the ListItem (game) already in line 6. So, it is good if different id is used here because ID's should be unique in an html page.